modules.json

```diff
@@ -130,7 +133,7 @@
                 "git_sha": "169b2b96c1167f89ab07127b7057c1d90a6996c7"
             },
             "gatk4/markduplicates": {
-                "git_sha": "169b2b96c1167f89ab07127b7057c1d90a6996c7"
+                "git_sha": "df2620cfc7e4c21b14ed03c1c928f09fbabf83c4"
```

Comment:
was that expected to be different?

Reply:
yeah module update. Someone changed the default ressource usage, which makes sense IMO